### ISSUE TITLE
docs: Issue on page Quick Start Guide to ScyllaDB Cloud

### ISSUE THREAD

**Issue by pogzie** [scylladb/scylladb-cloud-doc-issues#20]
I would like to report an issue on page https://cloud.docs.scylladb.com/master/scylladb-quickstart/index

### Problem
Information on "Allowed IPs" could get more information. Customer asked about where this value is used. Upon checking, the IPs are applied to Security Groups in the corresponding cloud provider

###  Suggest a fix
Add information on where this value is used.

**Comment by scylladb-buff** [comment]
> Whitelist IP addresses that will be able to communicate with your cluster on ScyllaDB Cloud to ensure the security of your data.
>
> (Recommended) Enable the Enable VPC Peering option to avoid routing your traffic over the open internet and ensure that your cluster and the application layer connect via a private network. Enabling this option will allow you to set up VPC peering after you launch the cluster.

Should be changed as follows:

> On the next page you will be asked to configure:
> 
> **Allowed CIDR ranges**.  [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) network ranges that can communicate with your cluster on ScyllaDB Cloud. 
You can specify a single IP as xxxx.xxxx.xxxx.xxxx/32.
To improve the security of your data, for public clusters, by default, clusters can be accessed by your current IP. You can change this as well by using this user interface.  
> 
> **Network Type** Allows you to choose between a Public internet connection and a Private connection
> - Public internet
> Will open the database to the internet. The database can be accessed from the CIDR network ranges defined in the Allowed CIDR ranges list.
> 
> - Private connection (Recommended) 
> Enable a private connection to avoid routing your traffic over the open internet and ensure that your cluster and the application layer connect via a private network connection. Enabling this option will allow you to set up VPC peering or Transit gateway connection for your application VPCs once you launch the cluster.
> ...

@annastuchlik Please format and align the tone with the rest of the documentation.

**Comment by scylladb-buff** [comment]
I just made more changes to the copy above to reflect the changes in the UI.

**Comment by annastuchlik** [comment]
You're right that the documentation should be extended. However, we should not provide such detailed explanations in the _Quick Start Guide_. I'll think about where to add it. Perhaps we'll use the new tooltip feature once the new version of the Sphinx theme is out.

For sure, I need to update the options, as some opinions are missing, especially _Public Internet_ vs _VPC Peering / Transit Gateway_ and _DB-level encryption._ 

From what I see, the option name is still _Allowed IPs_, not _Allowed CIDR ranges_, so I'll leave it.

**Comment by scylladb-buff** [comment]
**Allowed IP** is being changed to **Allowed CIDR ranges** in the UI this week as per https://github.com/scylladb/siren-frontend/issues/4850 
Could you make sure we coordinate the changes?
